s5_subwords/local/prepare_dict_subword.sh

```diff
@@ -47,7 +46,7 @@ echo -n "" >$dir/extra_questions.txt
 
 glossaries="<UNK> <sil>"
 
-if [ $stage -le 3 ]; then
+if [ $stage -le 0 ]; then
```

Comment:
If you want both of them to run at stage 0 why don't you combine them? And shouldn't the commands between the two if statements by inside the if statements? Or do you want them to run even when stage > 0?